File: 0896-monotonic-array/0896-monotonic-array.py
class Solution(object):
    def isMonotonic(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        increase=True
        decrease=True
    
        for i in range(1,len(nums)):
            if nums[i]<nums[i-1]:
                decrease=False
            if nums[i]>nums[i-1]:
                increase=False
        return increase or decrease          


        

File: 0896-monotonic-array/0896-monotonic-array.java
class Solution {
    public boolean isMonotonic(int[] nums) {
        boolean increase=true;
        boolean decrease=true;
        for(int i=1;i<nums.length;i++){
            if(nums[i]<nums[i-1]){
                decrease=false;
            }
            if(nums[i]>nums[i-1]){
                increase=false;
            }
        }
        return increase||decrease;

    }
}